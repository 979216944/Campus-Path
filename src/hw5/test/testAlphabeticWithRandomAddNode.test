# Tests that listNodes list nodes in alphabetic order

# Create graph
CreateGraph g

# Add nodes to g in random order
AddNode g e
AddNode g c
AddNode g a
AddNode g d
AddNode g b

# Print nodes of g
ListNodes g