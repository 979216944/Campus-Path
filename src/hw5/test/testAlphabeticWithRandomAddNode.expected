# Tests that listNodes list nodes in alphabetic order

# Create graph
created graph g

# Add nodes to g in random order
added node e to g
added node c to g
added node a to g
added node d to g
added node b to g

# Print nodes of g
g contains: a b c d e